PyITA/test_ITA.py

```diff
@@ -0,0 +1,252 @@
+import pytest
```

Comment:
A short description of what this file does would be very nice. I assume it does test the newly added feature for GeLU. In this case, I would change the name as it is not a full test for the ITA golden model but rather a subset.